### PR TITLE
BO-97: Added memcache extension

### PR DESCRIPTION
**Pull request by dimitriyremerov**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,6 +23,9 @@ RUN docker-php-ext-install mcrypt \
       intl \
       pgsql
 
+RUN pecl install -f memcache
```
Comment (jaysphoto) [inline Dockerfile]: The `memcache` ext is quite old (deprecated), we should use `memcached` instead I think. Is there any reason to fall back on pecl ? docker-php-ext-install should be able to handle this as well.

Comment (dimitriyremerov) [inline Dockerfile]: `Possible values for ext-name:
bcmath bz2 calendar ctype curl dba dom enchant exif fileinfo filter ftp gd gettext gmp hash iconv imap interbase intl json ldap mbstring mcrypt mssql mysql mysqli oci8 odbc opcache pcntl pdo pdo_dblib pdo_firebird pdo_mysql pdo_oci pdo_odbc pdo_pgsql pdo_sqlite pgsql phar posix pspell readline recode reflection session shmop simplexml snmp soap sockets spl standard sybase_ct sysvmsg sysvsem sysvshm tidy tokenizer wddx xml xmlreader xmlrpc xmlwriter xsl zip`

Comment (jaysphoto) [inline Dockerfile]: okay it's not supported apparently :)

then, could we do:
`RUN cd /tmp && git clone https://github.com/php-memcached-dev/php-memcached.git && cd php-memcached && phpize && ./configure && make && make install` ?

pecl is not a nice dependency to have afaik, and there's no guarantee that it's maintained properly. Maybe @silvanu also has some experience with this, from the Unitt cookbooks.

**Comment by jaysphoto** [comment]
oh, if you add a PHP extension, please update the `composer.json`, add it as a requirement:

```
require: {
  "ext-memcached": "*"
}
```

Heroku (etc.) will pick this up and make sure that the extension is activated.

File: Dockerfile (hunk 1 above)
Comment (silvanu) [inline Dockerfile]: Better to use `memcached`. Pecl will use the same source `https://github.com/php-memcached-dev/php-memcached`. I don't see any problem in working with Pecl.

Comment (dimitriyremerov) [inline Dockerfile]: I changed it to `memcached` (via pecl).

**Comment by dimitriyremerov** [comment]
@jaysphoto @silvanu Can we merge it now?

**Comment by jaysphoto** [comment]
:octocat:

**Pull request merged**